bootloaders/riotboot/README.md

```diff
@@ -108,3 +98,43 @@ To test building, flashing and booting the first slot:
 For the second slot:
 
 `BOARD=samr21-xpro APP_VER=$(date +%s) make -C tests/riotboot/ riotboot/flash-slot1 test`
+
+# Quick riotboot porting guide
+
+Your board is not supported yet? Try to port riotboot!
+
+## Porting to a board based on am Arm Cortex-M0+/3/4/7 MCU
+
+Extending riotboot to support another board with a Cortex-M0+/3/4/7
+microcontroller is rather straightforward. You need to:
+
+  - Provide the variables `ROM_START_ADDR` and `ROM_LEN` as well as
```

Comment:
Here and below, there's no need for spaces before `-` since there are first level bullet points.